src/hidimstat/distilled_conditional_randomization_test.py

```diff
@@ -69,25 +61,27 @@ class D0CRT(BaseVariableImportance):
         Whether to use scaled statistics when computing importance.
     random_state : int, default=2022
         Random seed for reproducibility.
+    reuse_screening_model: bool, default=True
```

Comment:
From what I understand this was hidden, ie, the screening weights were used when screening=True

Reply:
If a non-linear model is used for y-distillation, and screening is performed.

Reply:
But the screening seems to be based on (expensive to compute) Lasso coefficients... It would be a bit silly to do screening with a Lasso and then used another learner. But I agree that a separate issue/PR can be opened for that.

Reply:
I opened the issue #378

Reply:
see PR #380